storage/persistent_storage/persistent-storage-azure-file.adoc

```diff
@@ -30,9 +30,9 @@ Azure File volumes use Server Message Block.
 
 [IMPORTANT]
 ====
-In future {product-title} versions, volumes provisioned using existing in-tree plugins are planned for migration to their equivalent CSI driver. CSI automatic migration should be seamless. Migration does not change how you use all existing API objects, such as persistent volumes, persistent volume claims, and storage classes. For more information about migration, see xref:../../storage/container_storage_interface/persistent-storage-csi-migration.adoc#persistent-storage-csi-migration[CSI automatic migration].
+{product-title} 4.13 and later provides automatic migration for the Azure File in-tree volume plugin to its equivalent CSI driver.
 
-After full migration, in-tree plugins will eventually be removed in future versions of {product-title}.
+CSI automatic migration should be seamless. Migration does not change how you use all existing API objects, such as persistent volumes, persistent volume claims, and storage classes. For more information about migration, see xref:../../storage/container_storage_interface/persistent-storage-csi-migration.adoc#persistent-storage-csi-migration[CSI automatic migration].
 ====
```

Comment:
LGTM.

For reference: https://issues.redhat.com/browse/STOR-737

Reply:
Actually, shouldn't this be targetting the 4.13 branch?

Reply:
LGTM and yes applies to 4.13+

Reply:
@bertinatto @gcharot this fix is targeted at 4.13+. OK?

Reply:
yes correct, thanks